python/templates/model.py.jinja

```diff
@@ -1,5 +1,5 @@
 {% set class_name = model.class_info.name %}
-{% if model.required_properties == [] and model.optional_properties == [] and False %}
+{% if model.required_properties == [] and model.optional_properties == [] %}
```

Comment:
The revert looks correct, the `and False` looks silly and that it'll break stuff, so no wonder if broke stuff.